Scripts/DCS-BIOS/lib/modules/aircraft_modules/FW-190D9.lua

```diff
@@ -0,0 +1,224 @@
+module("FW-190D9", package.seeall)
+
+local Module = require("Module")
+
+--- @class FW_190A8: Module
+local FW_190A8 = Module:new("FW-190D9", 0x3400, { "FW-190D9" })
```

Comment:
Variable should probably be named FW_190D9?

Reply:
of course

Reply:
fixed